tests/test_appsettings.py

```diff
@@ -425,6 +440,20 @@ def test_nested_dict_setting_required_both_inner_and_outer_setting(self):
             assert len(outer_setting.value.items()) == 1
             assert outer_setting.value.get("inner_setting") == "Value"
 
+    def test_file_setting_string(self):
+        setting = appsettings.FileSetting(name="file")
+        with override_settings(FILE=self.temporary_file_name):
+            setting.check()
+            assert isinstance(setting.value, Path)
+            assert setting.value.samefile(self.temporary_file_name)
+
+    def test_file_setting_path(self):
+        setting = appsettings.FileSetting(name="file")
+        with override_settings(FILE=Path(self.temporary_file_name)):
+            setting.check()
+            assert isinstance(setting.value, Path)
+            assert setting.value.samefile(self.temporary_file_name)
```

Comment:
I'd create a separate test case for the `FileSetting`. It seems the `SettingTestCase` have grown too big already.

Reply:
There are only two tests for `FileSetting`, it is probably the last setting to separate. I would separate all the tests for each setting type in another ticket/branch.

Reply:
We should do that as well. But addition of a new tests is great place to start with separation.

Reply:
Okey, I will do it, but it seems to be so strange to me. At this point there is one huge class `SettingTestCase` and one poor little class `FileSettingTestCase` with two tests.

Much better and cleaner would be to do it all at once, by my opinion.